src/glmtools.jl

```diff
@@ -276,6 +294,19 @@ function inverselink(nbl::NegativeBinomialLink, η::Real)
     return μ, deriv, oftype(μ, NaN)
 end
 
+linkfun(pl::PowerLink, μ::Real) = pl.λ == 0 ? log(μ) : μ^pl.λ
+linkinv(pl::PowerLink, η::Real) = pl.λ == 0 ? exp(η) : η^(1 / pl.λ)
+function mueta(pl::PowerLink, η::Real)
+    if pl.λ == 0
+        return exp(η)
+    else
+        invλ = inv(pl.λ)
+        return invλ * η^(invλ - 1)
+    end
+end
+inverselink(pl::PowerLink, η::Real) =
+    linkinv(pl, η), mueta(pl, η), oftype(float(η), NaN)
```

Comment:
This should be more efficient as it avoids computing `exp(η)` and `inv(pl.λ)` twice (I hope it's correct, anyway you get the idea):
```suggestion
function inverselink(pl::PowerLink, η::Real)
    if pl.λ == 0
        μ = exp(η)
        return μ, μ, oftype(float(η), NaN)
    else
        invλ = inv(pl.λ)
        return η^invλ, invλ * η^(invλ - 1), oftype(float(η), NaN)
    end
```

Reply:
Committed the suggestions after minor changes.